public/locales/en.json

```diff
@@ -246,7 +246,9 @@
             "keymap": "Keymap",
             "tabCharacter": "Tab character",
             "tabSize": "Tab size",
-            "spellChecker": "Spell checking"
+            "spellChecker": "Spell checking",
+            "pdf": "PDF export is unavailable.",
+            "why": "Why?"
```

Comment:
This shouldn't be under "preferenes"

Reply:
Put it to "export"

Reply:
This should have been under 'menu', but apparently I rebased it wrong. Now it's under 'export' as requested